packages/design-system-docs/src/pages/styles/typography/_typography.docs.scss

```diff
@@ -26,6 +26,37 @@ Markup: typography-body.example.html
 Style guide: styles.typography.body
 */
 
+/*
+List
+
+Lists organize written information for users.
+
+Markup: list.example.html
+
+### When to use
+
+- Use an ordered list when you need to display text in some ranking, hierarchy, or series of steps.
+- Use unordered lists to display text in no specific order.
+
+### When to consider alternatives
+
+- If you need to communicate long lists of narrative text.
+
+### Usage
+
+- Use sentence case and begin lists with a capital letter.
+- Use punctuation appropriate to the text. Do not leave sentences without periods.
+
+### Accessibility
+- When a list has a heading, consider adding an `id` attribute to the heading element, and an `aria-labelledby` attribute to the list element. Screen readers will then announce the heading text when users navigate to the list.
```

Comment:
This advice is new to me. I'm not necessarily debating it, but I'm curious about its origin.

Reply:
I'm unsure of the origin of this - there might be some info in the depths of closed PRs